src/routes/DigitalAssets/routes/Receive/DigitalAssetsReceiveView.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Link } from 'react-router'
+
+import { JText, JIcon, JRaisedButton } from 'components/base'
+import { AddressPicker } from 'components'
+import { clipboard, qrCode } from 'services'
+
+const qrAppearance = {
+  size: 240,
+  errorCorrectionLevel: 'high',
```

Comment:
we have [config option for qr code](https://github.com/jibrelnetwork/jwallet-web/blob/develop/src/config.js#L38)
please use (update if necessary) it

Reply:
Yeah, I saw the config, but I thought, I was needed extend it, not update )